src/components/LocationDetails.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactTooltip from "react-tooltip";
+import Collapsible from "react-collapsible";
+
+function LocationDetails(props) {
+  const featureProps = props.feature.properties;
+
+  const propsMapping = [
+    {
+      title: 'Demographic Summary',
+      content: [
+        { id: 'persons_num', label: 'Population' },
+        { id: 'median_aud', label: 'Median Income', format: (val) => `AUD ${val}` },
+      ]
+    },
+    {
+      title: 'Economic Summary',
+      content: [
+        { id: 'quartile', label: 'Income Quartile' },
+        { id: 'inequality', label: 'Inequality (lower is better)', format: (val) => `${val}%` },
+        { id: null, label: 'Visitor time spent by quartile' },
+      ]
+    },
+    {
+      title: 'Growth Summary',
+      content: [
+        { id: 'income_diversity', label: 'GDP Growth Potential', desc: 'Economic growth is an increase in the production of economic goods and services,compared from one period of time to another... Traditionally, aggregate economic growth is measured in terms of gross national product (GNP) or gross domestic product (GDP), although alternative metrics are sometimes used.' },
+        { id: 'bridge_diversity', label: 'Job Resilience', desc: 'The ability to adjust to career change as it happens and,by extension, adapt to what the market demands.' },
+        { id: 'bsns_growth_rate', label: 'Business Growth Index', desc: 'The growth rate is the measure of a company’s increase in revenue and potential to expand over a set period.' },
+        { id: 'SA1_7DIGITCODE_LIST', label: 'Included SA1 Regions' },
+      ]
+    }
+  ];
+
+  const renderMetric = (metric) => {
+    let value = featureProps[metric.id];
+    if (typeof value === 'number') {
+      value = value > 10 ? Math.floor(value) : Math.floor(value * 10000) / 10000;
```

Comment:
Why floor all numbers?

Also, it's possible that some values which should be formatted as strings will be stored as numbers and vice versa. For instance, `"SA2_MAIN16": 401011001` should not run through any number formatters.

It might be scale better to have propsMapping's `format` be an enum/key for a switch statement here, so we can be explicit about every property's format without needing to duplicate formatting functions, e.g.
```js
const propsMapping = [
    ...
    { id: 'median_aud', label: 'Median Income', format: 'currency_aud' },
    ...
    { id: 'quartile', label: 'Income Quartile', format: 'number' },
    { id: 'inequality', label: 'Inequality (lower is better)' },
    ...
]
...
const renderMetric = (metric) => {
    ...
    switch (metric.format) {
        case 'number':
            // format value as number
        case 'currency_aud':
            // format as AUD
        default:
            // leave as raw value by default
    }
    ....
}
```